src/views/Correction.vue

```diff
@@ -408,60 +414,49 @@ export default {
     async fetchDraftFiling (): Promise<void> {
       const url = `businesses/${this.entityIncNo}/filings/${this.filingId}`
       await axios.get(url).then(res => {
-        if (res && res.data) {
-          const filing: any = res.data.filing
-          try {
```

Comment:
The "if" is redundant as checking the filing below will throw anyway.

The "try" is redundant because THIS code is already in the "then function" for the axios call, so any throws in here will be caught by the "catch function" below.